grc/gui/canvas/block.py

```diff
@@ -95,17 +96,39 @@ def rotation(self, rot):
         self.states['rotation'] = rot
 
     def _update_colors(self):
-        self._bg_color = (
-            colors.MISSING_BLOCK_BACKGROUND_COLOR if self.is_dummy_block else
-            colors.BLOCK_BYPASSED_COLOR if self.state == 'bypassed' else
-            colors.BLOCK_ENABLED_COLOR if self.state == 'enabled' else
-            colors.BLOCK_DISABLED_COLOR
-        )
+        def get_bg():
+            """
+            Get the background color for this block
+
+            Explicit is better than a chain of if/else expressions,
+            so this was extracted into a nested function.
+            """
+            c = colors
```

Comment:
I wonder if it is really worth saving 5 characters?

Reply:
fixed